e2e-tests/IIoTPlatform-E2E-Tests/Standalone/PublishNodesFromFileTestTheory.cs

```diff
@@ -18,59 +19,33 @@ namespace IIoTPlatform_E2E_Tests.Standalone {
     [TestCaseOrderer("IIoTPlatform_E2E_Tests.TestExtensions.TestOrderer", TestConstants.TestAssemblyName)]
     [Collection("IIoT Standalone Test Collection")]
     [Trait(TestConstants.TraitConstants.PublisherModeTraitName, TestConstants.TraitConstants.PublisherModeStandaloneTraitValue)]
-    public class PublishNodesFromFileTestTheory {
+    public class PublishSingleNodeStandaloneTestTheory {
         private readonly ITestOutputHelper _output;
         private readonly IIoTStandaloneTestContext _context;
 
-        public PublishNodesFromFileTestTheory(IIoTStandaloneTestContext context, ITestOutputHelper output) {
+        public PublishSingleNodeStandaloneTestTheory(IIoTStandaloneTestContext context, ITestOutputHelper output) {
             _output = output ?? throw new ArgumentNullException(nameof(output));
             _context = context ?? throw new ArgumentNullException(nameof(context));
             _context.OutputHelper = _output;
         }
 
         [Fact, PriorityOrder(1)]
-        public async Task Test_ReadSimulatedOpcUaNodes() {
-            var cts = new CancellationTokenSource(TestConstants.MaxTestTimeoutMilliseconds);
-            var simulatedOpcServer = await TestHelper.GetSimulatedOpcUaNodesAsync(_context, cts.Token);
-            Assert.NotNull(simulatedOpcServer);
-            Assert.NotEmpty(simulatedOpcServer.Keys);
-            Assert.NotEmpty(simulatedOpcServer.Values);
-        }
-
-        [Fact, PriorityOrder(2)]
         public async Task Test_CreateEdgeBaseDeployment_Expect_Success() {
             var cts = new CancellationTokenSource(TestConstants.DefaultTimeoutInMilliseconds);
             var result = await _context.IoTHubEdgeBaseDeployment.CreateOrUpdateLayeredDeploymentAsync(cts.Token);
-            _output.WriteLine("Created new EdgeBase layered deployment and publisher_standalone");
+            _output.WriteLine("Created/Updated new EdgeBase deployment");
             Assert.True(result);
         }
 
-        [Fact, PriorityOrder(3)]
-        public async Task Test_SwitchToStandaloneModeAndLoadPublishedNodesFile() {
-            var cts = new CancellationTokenSource(TestConstants.MaxTestTimeoutMilliseconds);
-            var simulatedOpcServer = await TestHelper.GetSimulatedOpcUaNodesAsync(_context, cts.Token);
-
-            //Create a published_nodes file with one node
-            var opcPlcServerNodes = simulatedOpcServer[simulatedOpcServer.Keys.First()];
-            var publishedNodes = opcPlcServerNodes;
-            publishedNodes.OpcNodes = opcPlcServerNodes.OpcNodes.Take(1).ToArray();
-            TestHelper.SavePublishedNodesFile(publishedNodes, _context);
-            _output.WriteLine("Saved published_nodes.json file");
-
-            await TestHelper.SwitchToStandaloneModeAsync(_context, cts.Token);
```

Comment:
Standalone test theories should start with switch to standalone mode, orchestrated mode is the default